boards/px4/fmu-v3/default.cmake

```diff
@@ -19,6 +19,7 @@ px4_add_board(
 		TEL4:/dev/ttyS6
 	DRIVERS
 		adc/board_adc
+		adc/ADS1115
```

Comment:
This is nitpicky, but for consistency how about keeping the folder name lowercase like all the other drivers and modules?

Reply:
Confirmed.